### DISCUSSION TITLE
IIS Nginx Deployment On same IP Address

### DISCUSSION THREAD

**Discussion by mabdalla123** [abpframework/abp#19351] [Q&A]
Iam trying to deploy an application in a subdomain ,  my limitations are that i cant use a separate ip address , so iam using the nginx of the Main Domain server , if a request is for the main domain then proceed as normal if the request is for the subdomain then it should point to the server that iam hosting the abp application on, how should i point to the abp Application , using the dns ? or ip address& port ? and if i used the ip& port should i change this on the appsetting ?

**Comment by maliming** [comment]
hi

> and if i used the ip& port should i change this on the appsetting ?

Yes. You need to update the domain info in `appsettings.json`.

See https://docs.abp.io/en/abp/latest/Deployment/Index

**Reply by mabdalla123** [reply]
Will I need to run the dbmigrator after changes?

**Reply by maliming** [reply]
Yes

**Reply by mabdalla123** [reply]
HI , so i have deployed the application , iam having  a problem with email  Settings 
this is the log 
2024-03-22 22:27:13.691 +02:00 [INF] Antiforgery token validation failed. The required antiforgery header value "RequestVerificationToken" is not present.
Microsoft.AspNetCore.Antiforgery.AntiforgeryValidationException: The required antiforgery header value "RequestVerificationToken" is not present.
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.ValidateRequestAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.Filters.ValidateAntiforgeryTokenAuthorizationFilter.OnAuthorizationAsync(AuthorizationFilterContext context)
i can manage users , roles without a problem 
this only happened after the deployment
